_posts/2020-01-16-spring-bean-life-cycle.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "The Lifecycle of Spring Beans"
+categories: [spring-boot]
+modified: 2020-01-16
+excerpt: "An in-depth look at the paging support provided by Spring Data for querying
+          Spring Web MVC controllers and Spring Data repositories."
+image:
+  auto: 0012-pages
+
+---
+
+## What are Spring Beans?
+ > Spring Beans are Java Objects or Instances which is created and managed by Spring container.
+
+Spring `@ApplicationContext` is responsible for creating a Spring bean and manage the 

```

Comment:
This sentence is merely repeating what you said in the quote above. Please elaborate on the role of the `ApplicationContext` (without the `@`, by the way :)). 2 or 3 sentences should be enough.

Like: "When we develop an application with Spring, we tell Spring to take control of our objects. Spring's ApplicationContext is then responsible for managing our objects. We can tell Spring to take control, by using the `@Component` annotation, for example: <code example>"